agent/engine/docker_container_engine.go

```diff
@@ -44,7 +44,13 @@ const (
 // Timelimits for docker operations enforced above docker
 const (
 	// ListContainersTimeout is the timeout for the ListContainers API.
-	ListContainersTimeout   = 10 * time.Minute
+	ListContainersTimeout = 10 * time.Minute
+	// LoadImageTimeout is the timeout for the LoadImage API. It's set
+	// to much lower value than pullImageTimeout as it involves loading
+	// image from either a file or STDIN
+	// calls involved.
```

Comment:
This is still needed. I'm not sure what this value should be yet. Will create an issue to track this.